src/System.Net.Http/src/System/Net/Http/CurlHandler/CurlHandler.cs

```diff
@@ -164,7 +164,7 @@ static CurlHandler()
             Interop.Http.CurlFeatures features = Interop.Http.GetSupportedFeatures();
             s_supportsSSL = (features & Interop.Http.CurlFeatures.CURL_VERSION_SSL) != 0;
             s_supportsAutomaticDecompression = (features & Interop.Http.CurlFeatures.CURL_VERSION_LIBZ) != 0;
-            s_supportsHttp2Multiplexing = (features & Interop.Http.CurlFeatures.CURL_VERSION_HTTP2) != 0 && Interop.Http.GetSupportsHttp2Multiplexing();
+            s_supportsHttp2Multiplexing = (features & Interop.Http.CurlFeatures.CURL_VERSION_HTTP2) != 0 && Interop.Http.GetSupportsHttp2Multiplexing() && !UseSingletonMultiAgent;
```

Comment:
It would be nice if it were clearer why this only applies to OSX 10.13. Reading the code, I have no idea why that's the effect of adding the `&& !UseSingletonMultiAgent` clause.

Reply:
What is Http2Multiplexing? Using Http2 and Http1.1 a the same time? Or does it just mean "HTTP/2 is supported"?

Reply:
https://bagder.gitbooks.io/http2-explained/en/part11.html

my understanding is that it would use some HTTP/2 features (like header compression) but it would not send multiple parallel requests over single connection. It would either open new one, or it would wait for previous one to finish. 
I don't know if observed behavior is bug in curl or  in our handler wrapper but with sunsetting curl handler I did not want to spent too much time.

Reply:
That makes sense, thanks.

Reply:
>but it would not send multiple parallel requests over single connection

This doesn't make any sense to me.  The whole point of HTTP/2 is to send multiple requests over a single connection.  What is the point of using HTTP/2 if it isn't going to do that?

Reply:
> This doesn't make any sense to me

Whether or not it makes sense, that's the behavior until you opt-in to the multiplexing... from that post: "As libcurl tries to maintain existing behaviors to a far extent, you need to enable HTTP/2 multiplexing for your application with the CURLMOPT_PIPELINING option. Otherwise it will continue using one request at a time per connection."